portals/knowledge-management/src/Pages/ViewArticle.vue

```diff
@@ -85,9 +94,23 @@
                 article.value = response.data.article;
                 loading.value = false;
                 portalViewCount.value = response.data.portal_view_count;
+                feedback.value = response.data.article.vote ? response.data.article.vote.is_helpful : null;
+                articalWasHelpfulFeatureFlag.value = response.data.article_was_helpful_feature_flag;
             },
         );
     }
+    async function toggleFeedback(type) {
+        if (feedback.value === type) {
+            feedback.value = null; // Unselect if the same button is clicked
+        } else {
+            feedback.value = type; // Select the clicked button
+        }
+        const { post } = consumer();
```

Comment:
This should be move somewhere else outside of this function, in case it is used again in this file. Like below `const route = useRoute();` or something like that.

Reply:
Done